docs/Makefile

```diff
@@ -1,16 +1,24 @@
-BUILDDIR    = build
-SOURCEDIR   = .
-SPHINXBUILD = sphinx-build
-SPHINXOPTS  = -a -n
+BUILDDIR     = build
+LINKCHECKDIR = $(BUILDDIR)/linkcheck
+SOURCEDIR    = .
+SPHINXBUILD  = sphinx-build
+SPHINXOPTS   = -a -n
 
-.PHONY: help clean
+.PHONY: help clean docs linkcheck
 
 help:
 	$(SPHINXBUILD) -M help $(SOURCEDIR) $(BUILDDIR) $(SPHINXOPTS)
 
 clean:
 	$(RM) -rv $(BUILDDIR)
 
+docs:
+	$(MAKE) linkcheck
+	$(MAKE) html
```

Comment:
A new target to build the docs _and_ run the link checker. The _Documentation_ page in the _Contributor Guide_ has been update to advise contributors to run `make docs` instead of `make html`, so link checking should be part of the normal workflow. CI is also being updated to run the same link checker.